server/battleships/utils/src/main/java/eu/optas/utils/GameState.java

```diff
@@ -0,0 +1,13 @@
+package eu.optas.utils;
+
+public enum GameState {
+        IN_PROGRESS("in-progress"),
```

Comment:
To label iš tiesų net nereikia, jei FE ir BE sutampa enum'ų pavadinimai tai gali tiesiog siųsdamas į FE pakviesti .toString()